bin/dist_get

```diff
@@ -14,63 +14,65 @@ check_writeable() {
 }
 
 download() {
-	local url="$1"
-	local output="$2"
+	dl_url="$1"
+	dl_output="$2"
 
-	if [ -z "$url" ] || [ -z "$output" ]; then
-		die "download takes exactly two arguments. was given '$@'"
-	fi
+	test "$#" -eq "2" || die "download requires exactly two arguments, was given $@"
 
 	if ! check_writeable "$output"; then
 		die "download error: cannot write to $output"
 	fi
 
 	if have_binary wget; then
-		printf 'Using wget to download "%s" to "%s"\n' "$url" "$output"
+		printf '==> Using wget to download "%s" to "%s"\n' "$url" "$output"
 		wget "$url" -O "$output"
 	elif have_binary curl; then
-		printf 'Using curl to download "%s" to "%s"\n' "$url" "$output"
+		printf '==> Using curl to download "%s" to "%s"\n' "$url" "$output"
 		curl --silent "$url" > "$output"
 	elif have_binary fetch; then
-		printf 'Using fetch to download "%s" to "%s"\n' "$url" "$output"
+		printf '==> Using fetch to download "%s" to "%s"\n' "$url" "$output"
 		fetch "$url" -o "$output"
 	else
 		die "no binary found to download $url. exiting."
 	fi
```

Comment:
You could also use `fi || return` instead of just `fi` above and it would do the right thing.